common/protob/messages-ethereum.proto

```diff
@@ -125,8 +129,9 @@ message EthereumTxAck {
  * @next Failure
  */
 message EthereumSignMessage {
-    repeated uint32 address_n = 1;  // BIP-32 path to derive the key from master node
-    required bytes message = 2;     // message to be signed
+    repeated uint32 address_n = 1;      // BIP-32 path to derive the key from master node
+    required bytes message = 2;         // message to be signed
+    optional bytes encoded_network = 3; // encoded Ethereum network, see protocol.md for details
```

Comment:
Should probably refer to `docs/common/ethereum-definitions.md` instead.

Reply:
[864ab42](https://github.com/trezor/trezor-firmware/pull/2410/commits/864ab42333e99a6b93de2707e299ef3f3fca06e6)